ukvm/gdb_signals.h

```diff
@@ -0,0 +1,65 @@
+/* Target signal numbers for GDB and the GDB remote protocol.
```

Comment:
This file and `gdb_signals.def` can't go in verbatim as it has GPL headers on it.

Can you rewrite the file and/or just add the specific `#defines` we need in a suitable place (e.g. `ukvm_module_gdb.c`)?